File: samples/server/petstore/haskell-servant/lib/Apis.hs
{-# LANGUAGE DataKinds #-}
{-# LANGUAGE TypeFamilies #-}
{-# LANGUAGE DeriveGeneric #-}
{-# LANGUAGE TypeOperators #-}
{-# LANGUAGE FlexibleInstances #-}
module Apis (
      api
    , API
    ) where

import Api.UserApi (UserApi)
import Api.PetApi (PetApi)
import Api.StoreApi (StoreApi)

import Data.Proxy
import Servant.API
import Test.QuickCheck
import qualified Data.Map as Map
import Utils

type API = UserApi :<|> PetApi :<|> StoreApi

api :: Proxy API
api = Proxy

File: samples/client/petstore/typescript-node/default/api/apis.ts
export * from './petApi';
import { PetApi } from './petApi';
export * from './storeApi';
import { StoreApi } from './storeApi';
export * from './userApi';
import { UserApi } from './userApi';
export const APIS = [PetApi, StoreApi, UserApi];
